setup.py

```diff
@@ -121,11 +121,43 @@ def _find(basepath, extension='.yml'):
             yield filename
 
 
+def _generate_country_file():
+    import json
+    from inspire_schemas.utils import COUNTRY_NAME_TO_CODE
+
+    schemas_js_dir = os.path.join(
+        os.path.dirname(__file__),
+        'schemas_js'
+    )
+    with open(os.path.join(schemas_js_dir, 'src', 'countries.json'), 'w+') as json_fd:
+        json.dump(list(COUNTRY_NAME_TO_CODE.keys()), json_fd)
```

Comment:
don't you want to serialize the json object you'll need directly, to avoid doing it in the JS code?
```suggestion
        json.dump([{"value": name} for name in COUNTRY_NAME_TO_CODE]), json_fd)
```

Reply:
Good idea!